### PR TITLE
Tech debt

### PR DESCRIPTION
**Pull request by AndrewShanks**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -113,95 +141,65 @@ microservice {
...
-  login = "http://localhost:9949/auth-login-stub/gg-sign-in"
-  loginContinue = "http://localhost:9000/business-account/corporation-tax"
-
-  paymentsFrontendBase = "http://localhost:9050/pay-online"
-
-  fileAReturn = "http://localhost:9030/cato"
-
-  mainPage = "http://localhost:9731/business-account/corporation-tax"
+toggles {
+  ct-payment-history = "true"
```
Comment (gianick) [inline conf/application.conf]:
```suggestion
  ct-payment-history = true
```

File: conf/application.conf [hunk 2]
```diff
@@ -113,95 +141,65 @@ microservice {
...
+  welsh-translation: true
```
Comment (gianick) [inline conf/application.conf]:
```suggestion
  welsh-translation = true
```

File: conf/application.conf [hunk 3]
```diff
@@ -153,55 +149,57 @@ mongodb {
...
-      makeCompanyDormant = "https://www.gov.uk/dormant-company/dormant-for-corporation-tax"
-      closeCompany = "https://www.gov.uk/closing-a-limited-company"
       certOfResidence = "https://www.gov.uk/guidance/get-a-certificate-of-residence"
+      closeCompany = "https://www.gov.uk/closing-a-limited-company"
+      makeCompanyDormant = "https://www.gov.uk/dormant-company/dormant-for-corporation-tax"
     }
 
     portal {
       host = "http://localhost:8080/portal"
+      activate = "/service/corporation-tax?action=activate&step=enteractivationpin"
```
Comment (suyangyu) [inline conf/application.conf]: move these two above the host

File: conf/application.conf [hunk 4]
```diff
@@ -153,55 +149,57 @@ mongodb {
...
+  fileAReturn = "http://localhost:9030/cato"
+
+  forms {
+    setupPartnership = "/forms/form/register-a-partner-or-a-partnership-for-self-assessment/new"
+  }
+
+  help-and-contact {
+    host = "http://localhost:9733"
+    home = "/business-account/help"
+    closeCompanyOrMakeDormant = "/business-account/help/corporation-tax/register-or-tell-hmrc-you-are-no-longer-trading"
```
Comment (suyangyu) [inline conf/application.conf]: move this above the host

**Pull request merged**